packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -4,26 +4,23 @@
 
 import 'dart:async';
 
-import 'package:test/src/executable.dart' as test; // ignore: implementation_imports
-
-import '../artifacts.dart';
 import '../base/common.dart';
 import '../base/file_system.dart';
 import '../base/io.dart';
 import '../base/logger.dart';
 import '../base/os.dart';
 import '../base/platform.dart';
 import '../base/process_manager.dart';
-import '../base/terminal.dart';
 import '../cache.dart';
-import '../dart/package_map.dart';
 import '../globals.dart';
 import '../runner/flutter_command.dart';
 import '../test/coverage_collector.dart';
-import '../test/flutter_platform.dart' as loader;
+import '../test/event_printer.dart';
+import '../test/runner.dart';
+import '../test/watcher.dart';
 
 class TestCommand extends FlutterCommand {
-  TestCommand() {
+  TestCommand({bool verboseHelp: false}) {
```

Comment:
flutter repo doesn't adhere to dartfmt since it doesn't meet our needs.  @Hixie has a list somewhere of all our wishes in a formatter...